### PR TITLE
Home Feed Tabs - Crayonsified

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

This PR introduces new styling for feed tabs. These will only be implemented on the home page for now. 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

https://d.pr/v/XrZd6T

## Added tests?

- [x] no, because they aren't needed

## Added to documentation?

- [x] no documentation needed

### CODE REVIEW

**Comment by nickytonline** [comment]
@ludwiczakpawel, I pushed a couple of commits.

File: app/assets/javascripts/initializers/initializeDrawerSliders.js [hunk 1]
```diff
@@ -1,57 +1,72 @@
+/* global swipeState InstantClick initializeSwipeGestures slideSidebar */
```
Comment (nickytonline) [inline app/assets/javascripts/initializers/initializeDrawerSliders.js:1]: This tells eslint that these functions are defined on the window in some other file.

**Comment by ludwiczakpawel** [comment]
@nickytonline thank you! i'll reping you once I update the arrows :) cheers!

**Comment by ludwiczakpawel** [comment]
the arrow should be fixed.

**Comment by nickytonline** [comment]
@ludwiczakpawel, I have latest of your branch and for some reason I can no longer switch themes. I'll wipe my app and rebuild. Maybe it was just a fluke.

**Comment by ludwiczakpawel** [comment]
@nickytonline hold on with wiping your app.. i can repro the issue. investigating...

**Comment by nickytonline** [comment]
> Awesome job, @ludwiczakpawel 👍
> 
> Small observation - I checked the homepage on mobile with `Ten X Hacker` theme; the `down` arrow on the Feed button does not show up (see screenshot).
> 
> Screen Shot 2020-05-20 at 8 14 35 AM

All sorted out now @msarit.

image

**Comment by ludwiczakpawel** [comment]
@msarit can you re-review?:)

**Pull request merged**